src/lib/components/json-schema/section/SchemaInputSection.tsx

```diff
@@ -1,35 +1,36 @@
 import { Button, Flex, Text, useDisclosure } from "@chakra-ui/react";
+import type { RJSFSchema } from "@rjsf/utils";
 import { observer } from "mobx-react-lite";
 
-import {
-  ViewSchemaButton,
-  AttachSchemaCard,
-  JsonSchemaDrawer,
-} from "lib/components/json-schema/";
+import { AttachSchemaCard } from "lib/components/json-schema/AttachSchemaCard";
 import { JsonSchemaForm } from "lib/components/json-schema/form";
+import { JsonSchemaDrawer } from "lib/components/json-schema/JsonSchemaDrawer";
+import { ViewSchemaButton } from "lib/components/json-schema/ViewSchemaButton";
 import { useSchemaStore } from "lib/providers/store";
 
 interface SchemaSectionProps {
+  type: "migrate" | "instantiate";
   codeHash: string;
   codeId: string;
   setSchemaInput: (input: string) => void;
 }
 
-export const SchemaSection = observer(
-  ({ codeHash, codeId, setSchemaInput }: SchemaSectionProps) => {
+export const SchemaInputSection = observer(
+  ({ type, codeHash, codeId, setSchemaInput }: SchemaSectionProps) => {
     const { getSchemaByCodeHash } = useSchemaStore();
     const jsonSchema = getSchemaByCodeHash(codeHash);
     const { isOpen, onClose, onOpen } = useDisclosure();
+    const prettyType = type[0].toUpperCase() + type.slice(1);
```

Comment:
use `capitalize` from our utils

Reply:
use `capitalize` from `lodash` instead. Removing our `capitalize` util in another PR